controlplane/cmd/monitor-client.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import (
+	"context"
+	"net"
+	"os"
+	"os/signal"
+	"syscall"
+	"time"
+
+	"github.com/golang/protobuf/ptypes/empty"
+	"github.com/ligato/networkservicemesh/controlplane/pkg/apis/crossconnect"
+	"github.com/sirupsen/logrus"
+	"google.golang.org/grpc"
+)
+
+func main() {
+	// Capture signals to cleanup before exiting
+	c := make(chan os.Signal, 1)
+	signal.Notify(c,
+		syscall.SIGHUP,
+		syscall.SIGINT,
+		syscall.SIGTERM,
+		syscall.SIGQUIT)
+	var err error
+	conn, err := dial(context.Background(), "tcp", "127.0.0.1:5007")
```

Comment:
Where do you intend to run this? Running this anywhere but within the pod exposing 5007 will not work.

Reply:
For the time being we are not going to run it inside the k8s cluster. Since we just take over these code, we intend to run it on the host machine. The goal is simple: a simple model to simulate events sending by MonitorServer, and extract information from the client.  
Maybe we should develop this functionality in another repository?  What's your advice?  
[#520](https://github.com/ligato/networkservicemesh/pull/520) is the same PR, I just move those file to another folder.